packages/gatsby-source-shopify/src/gatsby-node.js

```diff
@@ -56,9 +57,21 @@ export const sourceNodes = async (
     verbose = true,
     paginationSize = 250,
     includeCollections = [SHOP, CONTENT],
+    downloadImages = true,
+    defaultImageUrl = `https://raw.githubusercontent.com/mrhut10/gatsby/Feature/gatsby-source-shopify/downloadImagesOption/packages/gatsby-source-shopify/src/default.png`,
     shopifyQueries = {},
   }
 ) => {
+  // sending options into PluginOptions Singleton Class
+  PluginOptions.setShopName(shopName)
```

Comment:
I'm a bit iffy on this. Is there any big advantage over just passing an object around?

Reply:
yep thats cool, I can pull that out 👍

Reply:
i've pulled that out now :)